src/plugins/menu/menu.js

```diff
@@ -18,6 +18,11 @@ var componentName = "gcweb-menu",
 	justOpened,
 	isMobileMode, // Mobile vs Desktop
 	isMediumView,
+	preventFocusIn,
+	i18nInstruction = {
+		en: "Press the SPACEBAR to expand or the escape key to collapse this menu. Use the up and Down arrow keys to choose a submenu item. Press the Enter or Right arrow key to expand it, or the Left arrow or Escape key to collapse it. Use the up and Down arrow keys to choose an item on that level and the Enter key to access it.",
```

Comment:
For consistency purposes, I would suggest to put the letter "u" from "Up" in uppercase.

```suggestion
		en: "Press the SPACEBAR to expand or the escape key to collapse this menu. Use the Up and Down arrow keys to choose a submenu item. Press the Enter or Right arrow key to expand it, or the Left arrow or Escape key to collapse it. Use the Up and Down arrow keys to choose an item on that level and the Enter key to access it.",
```